routes/index.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   'get /agolworkers': 'getQueueCounts',
   'get /agol/:id': 'find',
   'get /agol/:id/*': 'setHostKey',
+  'put /agol/:id/*': 'setHostKey',
```

Comment:
"The PUT method requests that the enclosed entity be stored under the supplied Request-URI. If the Request-URI refers to an already existing resource, the enclosed entity SHOULD be considered as a modified version of the one residing on the origin server."
Meets the spec exactly.
http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html

I can set up post fallback pretty easily. Will update the PR with that.